src/components/flowchart/styles/_tooltip.scss

```diff
@@ -0,0 +1,63 @@
+@import './variables';
+
+.pipeline-flowchart__tooltip {
+  position: absolute;
+  top: -60px;
+  left: -20px;
+  padding: 12px 20px;
```

Comment:
Where do these numbers come from?

Reply:
From here. It's just the styling/positioning for the tooltip?

Reply:
Just seems like magic number positioning, what's it based off?

![magic](https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif)

Reply:
![image](https://user-images.githubusercontent.com/1155816/61880198-57333780-aeec-11e9-866d-d749b6338124.png)

Padding and array offset for the tooltip. It's just styling - if it's magic numbers then so is all CSS? 🤷‍♂